dist/uPlot.d.ts

```diff
@@ -164,6 +165,12 @@ type ArcV    = (p: Path2D, y: number, x: number, r: number, startAngle: number,
 type BezierCurveToH = (p: Path2D, bp1x: number, bp1y: number, bp2x: number, bp2y: number, p2x: number, p2y: number) => void;
 type BezierCurveToV = (p: Path2D, bp1y: number, bp1x: number, bp2y: number, bp2x: number, p2y: number, p2x: number) => void;
 
+type Sync = {
+	sub: (client: uPlot) => void;
+	unsub: (client: uPlot) => void;
+	pub: (type: string, client: uPlot, x: unknown, y: unknown, w: unknown, h: unknown, i: unknown) => void;
```

Comment:
these are all numbers.

they're event properties in the context of the emitting instance (client) and are similar to `e.clientX` of DOM events. `x` & `y` are left/top coordinates, `w` & `h` are the css pixel dimensions of the plotting area, `i` is the hovered index. not all events will have all arguments set, some args may be null.